### DISCUSSION TITLE
Reviung34 unibody support?

### DISCUSSION THREAD

**Discussion by MechHead777** [manna-harbour/miryoku#185] [Q&A] [answered]
Hello all! 

TLDR upfront: Reviung34 (unibody version) appears to not be listed as one of the supported 3x5_2 keymaps. Is it because its a unibody board? Do I just need to use like a 4x10 ortho instead? 

I am still very much a novice when it comes to this stuff so I apologize if this is the wrong place to post for help/guidance/support. I have flashed my Corne with Miryoku a few months ago and love it. When I went to do the same thing with my new reviung34, I was a bit bummed and frustrated that it wasn't working (qmk msys). Whenever I would attempt to compile the firmware, I would constantly get error messages. I've attached a .txt with the error message(s) I got. 

I think to add to the complexity, it appears that both the split and unibody versions of the reviung34 are within the same reviung34 folder. I believe the correct one is the 'default_rgb' for the unibody with all 1u keys. 

In my ignorance, I tried temporary relocating the split version keymap folders to another location. I tried copying the 5x3_2 keymap from the miryoku folder into the reviung34 keymap folder. I tried to flash the ferris sweep firmware onto the board. I tried to modify the .c files so that the thumb rows """lined up better""" between the reviung34 and miryoku. I went looking for gtips documentation on github but got a bunch of 404 pages.  I don't have hardly any experience with QMK other than flashing my corne with Miryoku (piece cake btw. great work!) and using QMK toolbox to flash via. 

Windows10 21H2. I installed qmk msys and """downloaded""" QMK within the last month. 

Please let me know if there is more info that I need to provide. Not sure if reviung34 can be added to the list of supported miryoku boards but it would be greatly appreciated. 

Thank you!!

[reviung34_error.txt](https://github.com/manna-harbour/miryoku/files/10346476/reviung34_error.txt)

**Comment by manna-harbour** [comment, marked as answer]
Here's a fix: https://github.com/qmk/qmk_firmware/pull/19499. For local builds you can use that PR branch. For workflow builds you can use `manna-harbour/qmk_firmware/community-layout-fix-pr` with the merge option. See https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#building. Here's a sample build run: https://github.com/manna-harbour/miryoku_qmk/actions/runs/3842910422.
